### PR TITLE
feat(reaction): add overload for reactionId

### PR DESCRIPTION
**Pull request by peterdeme**

### CODE REVIEW

**Comment by peterdeme** [comment]
@ferhatelmas ready to review ✅

File: src/Rest/RestClient.cs [hunk 1]
```diff
@@ -66,16 +66,16 @@ private HttpContent CreateFileStream(RestRequest request)
...
                 queryStringBuilder.Append(queryStringBuilder.Length == 0 ? "?" : "&");
-                queryStringBuilder.Append($"{p.Key}={Uri.EscapeDataString(p.Value.ToString())}");
+                queryStringBuilder.Append($"{p.Key}={Uri.EscapeDataString(p.Value)}");
             });
 
             return new Uri(_baseUrl, request.Resource + queryStringBuilder.ToString());
         }
 
-        public async Task<RestResponse> ExecuteHttpRequestAsync(RestRequest request)
+        internal async Task<RestResponse> ExecuteHttpRequestAsync(RestRequest request)
```
Comment (ferhatelmas) [inline src/Rest/RestClient.cs:78]: technically breaking change but nobody should have used it

File: src/Rest/RestRequest.cs [hunk 2]
```diff
@@ -11,22 +11,22 @@ internal RestRequest(string resource, HttpMethod method)
...
+        internal void AddHeader(string name, string value) => Headers[name] = value;
 
-        public void AddQueryParameter(string name, string value) => QueryParameters[name] = value;
+        internal void AddQueryParameter(string name, string value) => QueryParameters[name] = value;
 
-        public void SetJsonBody(string json) => JsonBody = json;
+        internal void SetJsonBody(string json) => JsonBody = json;
 
-        public void SetFileStream(System.IO.Stream stream, string name, string contentType)
+        internal void SetFileStream(System.IO.Stream stream, string name, string contentType)
```
Comment (ferhatelmas) [inline src/Rest/RestRequest.cs:29]: same here

File: src/Rest/RestClient.cs (hunk 1 above)
Comment (peterdeme) [reply src/Rest/RestClient.cs:78]: **the class was already internal** so no one could use it before. actually, I don't even understand this:

```csharp
internal class Myclass
{
  public void MyMethod()
  {
  }
}
```

File: src/Rest/RestRequest.cs (hunk 2 above)
Comment (peterdeme) [reply src/Rest/RestRequest.cs:29]: **the class was already internal** so no one could use it before. actually, I don't even understand this:

```csharp
internal class Myclass
{
  public void MyMethod()
  {
  }
}
```

**Review by ferhatelmas — APPROVED** [review]
LGTM, awesome 👍🏻

File: src/Rest/RestRequest.cs (hunk 2 above)
Comment (ferhatelmas) [reply src/Rest/RestRequest.cs:29]: oops then doesn't make sense, actually compiler should spit some warnings for it

**Pull request merged**